Test failures on s390x on Alpine

Hello there,

I [packaged universal-ctags][alpine commit] for [Alpine Linux][alpine website]. Alpine uses [musl libc][musl website] instead of glibc and supports quite a lot of architectures. Except for the `main/input-encoding-option.d` and `main/output-encoding-option.d` tests, universal-ctags builds (and tests) fine on Alpine. *The two tests likely fail due to incompatibilities with musl's iconv, building with `--disable-iconv` works as a workaround.*

Unfortunately, the `make check` command fails on the s390x architecture for current git HEAD (f42b573fea3b92ba58e3b4ca488ae013bcc9d758). The `make check` output is shown below. Most tests seem to fail due to an unexpected warning:

```
-ctags: Warning: ^[     ]*([a-zA-Z][a-zA-Z0-9_]*)[      ]*\{: regexp missing name pattern
```

A tarball containing the various `*-diff.txt` files can be downloaded here: [ctags-s390x-test-diffs.tar.gz](https://github.com/universal-ctags/ctags/files/3950225/ctags-s390x-test-diffs.tar.gz). Unfortunately, I didn't have time to investigate this further so far. I just wanted to documented this issue. If you need any additional information let me know.

```
REPOINFO   main/repoinfo.h
  CCLD     ctags
  RUN      tmain

Testing alias-for-unknown-language
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m
exit                                                        [32mpassed[39m

Testing allow-null-tag
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m
exit                                                        [32mpassed[39m

Testing broken-tagname-in-ectags-format
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m
exit                                                        [32mpassed[39m

Testing broken-tagname-quiet
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m
exit                                                        [32mpassed[39m

Testing case-insensitive-pattern
------------------------------------------------------------
[33mskipped[39m (feature "case-insensitive-filenames" is not available in /home/alpine/src/ctags/./ctags)

Testing broken-tagname
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m
exit                                                        [32mpassed[39m

Testing abnormal-output-file-names
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m

Testing c-anon-counter
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m
exit                                                        [32mpassed[39m

Testing clear-aliases
------------------------------------------------------------
stdout                                                      [32mpassed[39m
exit                                                        [32mpassed[39m

Testing dot-ctags-with-indentation
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m
exit                                                        [32mpassed[39m

Testing emacs-modeline-downcased-mode
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m

Testing emacs-modeline-non-alnum-in-tail
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing dynamic-kinds
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m
exit                                                        [32mpassed[39m

Testing emacs-modeline-non-alnum-in-head
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing emacs-modeline-non-alnum-in-head2
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing emacs-modeline-uppercased-marker
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m

Testing disable-fixed-field
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m
exit                                                        [32mpassed[39m

Testing enable-non-existing-kind
------------------------------------------------------------
stderr                                                      [32mpassed[39m

Testing enable-kind-postfix
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m

Testing excmd-combine-backward
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing cxx-how-kinds-defs-are-shared-or-copyed
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing enable-kind-prefix
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m

Testing excmd-combine
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing e-ctags-output
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing fixed-field-handling
------------------------------------------------------------
[33mskipped[39m (feature "json" is not available in /home/alpine/src/ctags/./ctags)

Testing extra-field
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m
exit                                                        [32mpassed[39m

Testing filter-option-write-to-file
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m

Testing extension-vs-pattern
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m
exit                                                        [32mpassed[39m

Testing filter-option
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing gcov-sandbox
------------------------------------------------------------
[33mskipped[39m (feature "sandbox" is not available in /home/alpine/src/ctags/./ctags)

Testing enable-kind-postfix-with-wildcard
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [31mfailed[39m (diff: /home/alpine/src/ctags/Tmain/enable-kind-postfix-with-wildcard.d/stderr-diff.txt)

Testing generate-anon-ids
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing interactive-mode
------------------------------------------------------------
[33mskipped[39m (feature "interactive" is not available in /home/alpine/src/ctags/./ctags)

Testing enable-kind-prefix-with-wildcard
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [31mfailed[39m (diff: /home/alpine/src/ctags/Tmain/enable-kind-prefix-with-wildcard.d/stderr-diff.txt)

Testing force-initializing-option
------------------------------------------------------------
stderr                                                      [31mfailed[39m (diff: /home/alpine/src/ctags/Tmain/force-initializing-option.d/stderr-diff.txt)

Testing interactive-option-write-to-file
------------------------------------------------------------
[33mskipped[39m (feature "interactive" is not available in /home/alpine/src/ctags/./ctags)

Testing interactive-resource-management
------------------------------------------------------------
[33mskipped[39m (feature "json" is not available in /home/alpine/src/ctags/./ctags)

Testing broken-fielddef
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m

Testing json-output-for-broken-input
------------------------------------------------------------
[33mskipped[39m (feature "json" is not available in /home/alpine/src/ctags/./ctags)

Testing json-output-format
------------------------------------------------------------
[33mskipped[39m (feature "json" is not available in /home/alpine/src/ctags/./ctags)

Testing disable-languages
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m

Testing json-output-much-parser-fields
------------------------------------------------------------
[33mskipped[39m (feature "json" is not available in /home/alpine/src/ctags/./ctags)

Testing json-output-to-file
------------------------------------------------------------
[33mskipped[39m (feature "json" is not available in /home/alpine/src/ctags/./ctags)

Testing json-ptag-in-list-extras
------------------------------------------------------------
[33mskipped[39m (feature "json" is not available in /home/alpine/src/ctags/./ctags)

Testing broken-extradef
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m

Testing invalid-encoding-option
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m
exit                                                        [32mpassed[39m

Testing kind-long-lregex
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing kind-long
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing broken-langdef
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m

Testing kinds-all-with-spec
------------------------------------------------------------
stderr                                                      [32mpassed[39m
exit                                                        [32mpassed[39m

Testing list-excludes
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing language-names
------------------------------------------------------------
exit                                                        [32mpassed[39m

Testing list-aliases
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing langmap-option
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m
exit                                                        [32mpassed[39m

Testing languages-and-language-force-options
------------------------------------------------------------
stderr                                                      [32mpassed[39m
exit                                                        [32mpassed[39m

Testing kind-abnormal-spec
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m

Testing list-kinddef-flags
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing list-language-fields
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing langmap-option-including-patterns
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m
exit                                                        [32mpassed[39m

Testing kind-and-role-defaults
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing list-mline-regex-flags
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing list-fields-with-prefix
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing json-limit-recursion
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m
exit                                                        [32mpassed[39m

Testing list-map-extensions
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing list-kinds-full
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m
exit                                                        [32mpassed[39m

Testing list-map-patterns
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing list-mtable-regex-flags
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing list-regex-flags
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing list-subparsers-all
------------------------------------------------------------
[33mskipped[39m (feature "xpath" is not available in /home/alpine/src/ctags/./ctags)

Testing list-pseudo-tags
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m
exit                                                        [32mpassed[39m

Testing load-conf-files-under-cwd-no-dot
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m
exit                                                        [32mpassed[39m

Testing extras-long
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing load-conf-files-under-cwd
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m
exit                                                        [32mpassed[39m

Testing load-conf-files-under-home
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m
exit                                                        [32mpassed[39m

Testing load-dot-ctags-twice
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m
exit                                                        [32mpassed[39m

Testing lregex-list-kinds-uniquely
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing list-extras
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [31mfailed[39m (diff: /home/alpine/src/ctags/Tmain/list-extras.d/stderr-diff.txt)
exit                                                        [32mpassed[39m

Testing lregex-clear-patterns
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m

Testing errors-about-parser-own-extras
------------------------------------------------------------
stderr                                                      [31mfailed[39m (diff: /home/alpine/src/ctags/Tmain/errors-about-parser-own-extras.d/stderr-diff.txt)

Testing lregex-list-kinds-full
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m
exit                                                        [32mpassed[39m

Testing lregex-list-kinds
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m
exit                                                        [32mpassed[39m

Testing lregex-reject-reserved-kind
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m
exit                                                        [32mpassed[39m

Testing lregex-kind-letter
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m

Testing map-for-unknown-language
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m
exit                                                        [32mpassed[39m

Testing mline-no-advance
------------------------------------------------------------
stderr                                                      [32mpassed[39m

Testing maxdepth
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing map-lang-option
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m
exit                                                        [32mpassed[39m

Testing mtable-avoid-infinite-loop-at-the-eof
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m

Testing mtable-pinning
------------------------------------------------------------
stderr                                                      [32mpassed[39m

Testing multi-roles
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing nameless-long-option
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m
exit                                                        [32mpassed[39m

Testing nested-mio
------------------------------------------------------------
stdout                                                      [32mpassed[39m
exit                                                        [32mpassed[39m

Testing nested-subparsers-multilines
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing mtable-stats
------------------------------------------------------------
stdout                                                      [31mfailed[39m (diff: /home/alpine/src/ctags/Tmain/mtable-stats.d/stdout-diff.txt)
stderr                                                      [32mpassed[39m

Testing kinddef
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m

Testing omit-long-patterns-etags
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m

Testing omit-long-patterns
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing list-roles-with-kind-names
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing list-params
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [31mfailed[39m (diff: /home/alpine/src/ctags/Tmain/list-params.d/stderr-diff.txt)

Testing map-removing
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m
exit                                                        [32mpassed[39m

Testing null-description-in-pseudo-tag
------------------------------------------------------------
stdout                                                      [32mpassed[39m
exit                                                        [32mpassed[39m

Testing option-dump-options
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing lregex-unexpected-regex-kind
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m

Testing option-use-slash-as-filename-separator
------------------------------------------------------------
[33mskipped[39m (feature "win32" is not available in /home/alpine/src/ctags/./ctags)

Testing mtable-mutual-entering
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m

Testing option-echo-and-force-quit
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m
exit                                                        [32mpassed[39m

Testing no-input-encoding-option
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m
exit                                                        [32mpassed[39m

Testing list-fields
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [31mfailed[39m (diff: /home/alpine/src/ctags/Tmain/list-fields.d/stderr-diff.txt)
exit                                                        [32mpassed[39m

Testing option-totals-extra
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing options-directory
------------------------------------------------------------
stderr                                                      [32mpassed[39m
exit                                                        [32mpassed[39m

Testing list-roles
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [31mfailed[39m (diff: /home/alpine/src/ctags/Tmain/list-roles.d/stderr-diff.txt)
exit                                                        [32mpassed[39m

Testing option-dump-keywords
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing options-maybe
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m
exit                                                        [32mpassed[39m

Testing optlib-dir-option
------------------------------------------------------------
stderr                                                      [32mpassed[39m

Testing parser-init-useCork
------------------------------------------------------------
stdout                                                      [31mfailed[39m (diff: /home/alpine/src/ctags/Tmain/parser-init-useCork.d/stdout-diff.txt)

Testing output-format-option
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing parser-own-fields-clearing-all
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing output-input-field-with-no-escape
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing lregex-kind-name
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m

Testing optlib-message-flag
------------------------------------------------------------
stdout                                                      [31mfailed[39m (diff: /home/alpine/src/ctags/Tmain/optlib-message-flag.d/stdout-diff.txt)
stderr                                                      [32mpassed[39m

Testing pretend-option
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing ptag-kind-desc
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing parser-own-fields
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m

Testing readtags-broken-input
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m
exit                                                        [32mpassed[39m

Testing readtags-qualifier-end-field
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m
exit                                                        [32mpassed[39m

Testing readtags-combine
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m
exit                                                        [32mpassed[39m

Testing sandbox-default-req
------------------------------------------------------------
[33mskipped[39m (feature "sandbox" is not available in /home/alpine/src/ctags/./ctags)

Testing regex-patterns-from-file
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m

Testing sandbox-crash
------------------------------------------------------------
[33mskipped[39m (feature "debug" is not available in /home/alpine/src/ctags/./ctags)

Testing regex-for-unknown-language
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m
exit                                                        [32mpassed[39m

Testing readtags-qualifier
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m
exit                                                        [32mpassed[39m

Testing sandbox-unknown-submode
------------------------------------------------------------
[33mskipped[39m (feature "sandbox" is not available in /home/alpine/src/ctags/./ctags)

Testing sandbox-no-eager-guessing
------------------------------------------------------------
[33mskipped[39m (feature "sandbox" is not available in /home/alpine/src/ctags/./ctags)

Testing sandbox
------------------------------------------------------------
[33mskipped[39m (feature "sandbox" is not available in /home/alpine/src/ctags/./ctags)

Testing sandbox-with-eager-guessing
------------------------------------------------------------
[33mskipped[39m (feature "sandbox" is not available in /home/alpine/src/ctags/./ctags)

Testing pattern-length-limit
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing selector-xml-root-elelement
------------------------------------------------------------
[33mskipped[39m (feature "xpath" is not available in /home/alpine/src/ctags/./ctags)

Testing sorted-help-message
------------------------------------------------------------
[33mskipped[39m (GNU sort is needed to run this test case)

Testing option-extras-enabling-all
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [31mfailed[39m (diff: /home/alpine/src/ctags/Tmain/option-extras-enabling-all.d/stderr-diff.txt)
exit                                                        [32mpassed[39m

Testing sorted-list-languages
------------------------------------------------------------
[33mskipped[39m (GNU sort is needed to run this test case)

Testing selector-dont-select-disabled-lang
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m
exit                                                        [32mpassed[39m

Testing output-file-resource-management
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing scopesep
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing tag-relative-option-no-optarg
------------------------------------------------------------
exit                                                        [32mpassed[39m

Testing tmain-skip-example
------------------------------------------------------------
[33mskipped[39m (example: no such feature)

Testing two-inputs-requiring-scope-stack
------------------------------------------------------------

Testing tag-relative-option
------------------------------------------------------------
[33mskipped[39m (--tag-relative=always|never is not available on this platform)

Testing w32-slash-in-exclude-option
------------------------------------------------------------
[33mskipped[39m (feature "win32" is not available in /home/alpine/src/ctags/./ctags)

Testing tab-in-parser-specific-field
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing trace-option
------------------------------------------------------------
[33mskipped[39m (feature "debug" is not available in /home/alpine/src/ctags/./ctags)

Testing tag-relative-option-in-etags
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing tmain-example
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m
exit                                                        [32mpassed[39m
tags                                                        [32mpassed[39m

Testing warn-reusing-kind-letter
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing wrong-parser-name-in-field
------------------------------------------------------------
stderr                                                      [32mpassed[39m
exit                                                        [32mpassed[39m

Testing subparser-direction
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing xref-output-to-file
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing wildcard-in-lang-of-fields-option
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing parser-own-extras
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing nested-subparsers
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing tags-pseudo-tags
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [31mfailed[39m (diff: /home/alpine/src/ctags/Tmain/tags-pseudo-tags.d/stderr-diff.txt)
exit                                                        [32mpassed[39m

Testing xformat-and-parser-own-field
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [31mfailed[39m (diff: /home/alpine/src/ctags/Tmain/xformat-and-parser-own-field.d/stderr-diff.txt)

Testing roledef
------------------------------------------------------------
stdout                                                      [32mpassed[39m
stderr                                                      [32mpassed[39m

Testing xref-ptag-in-list-extras
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Testing xformat-common-fields
------------------------------------------------------------
stdout                                                      [32mpassed[39m

Failed tests
============================================================
enable-kind-postfix-with-wildcard/stderr-compare
enable-kind-prefix-with-wildcard/stderr-compare
force-initializing-option/stderr-compare
list-extras/stderr-compare
errors-about-parser-own-extras/stderr-compare
mtable-stats/stdout-compare
list-params/stderr-compare
list-fields/stderr-compare
list-roles/stderr-compare
parser-init-useCork/stdout-compare
optlib-message-flag/stdout-compare
option-extras-enabling-all/stderr-compare
tags-pseudo-tags/stderr-compare
xformat-and-parser-own-field/stderr-compare

make: *** [Makefile:5632: tmain] Error 1
```


[alpine commit]: https://git.alpinelinux.org/aports/commit/?id=a92e43efbc78b4f7a6b601653f07fb80e1ebd25f
[alpine website]: https://alpinelinux.org/
[musl website]: https://www.musl-libc.org/

--- a/main/lregex.c
+++ b/main/lregex.c
@@ -478,7 +478,7 @@ static flagDefinition prePtrnFlagDef[] = {
 static void scope_ptrn_flag_eval (const char* const f  CTAGS_ATTR_UNUSED,
 				  const char* const v, void* data)
 {
-	unsigned long *bfields = data;
+	unsigned int *bfields = data;
 
 	if (strcmp (v, "ref") == 0)
 		*bfields |= SCOPE_REF;
@@ -497,7 +497,7 @@ static void scope_ptrn_flag_eval (const char* const f  CTAGS_ATTR_UNUSED,
 static void placeholder_ptrn_flag_eval (const char* const f  CTAGS_ATTR_UNUSED,
 				     const char* const v  CTAGS_ATTR_UNUSED, void* data)
 {
-	unsigned long *bfields = data;
+	unsigned int *bfields = data;
 	*bfields |= SCOPE_PLACEHOLDER;
 }
 

